src/System.Collections/tests/Generic/Dictionary/Dictionary.Generic.Tests.ConcurrentAccessDetection.netcoreapp.cs

```diff
@@ -0,0 +1,125 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Generic.Dictionary
+{
+    public class DictionaryConcurrentAccessDetectionTests
+    {
+        async Task DictionaryConcurrentAccessDetection<TKey, TValue>(Dictionary<TKey, TValue> dictionary, bool isValueType, object comparer, Action<Dictionary<TKey, TValue>> add, Action<Dictionary<TKey, TValue>> get, Action<Dictionary<TKey, TValue>> remove, Action<Dictionary<TKey, TValue>> removeOutParam)
+        {
+            Task task = Task.Factory.StartNew(() =>
+            {
+                //break internal state
```

Comment:
No need to fix unless you reset the PR again...we use single space after //
Also "break internal state" could be more explicit eg " set an empty entries array to simulate a dictionary corrupted by concurrent access". 
Am I reading that right? I was assuming you would set an entries array with a loop, how does an empty one work?

Reply:
Ah, I see (now I'm not reading on the phone). The correct Bucket points to an Entry with .next of 0, so it loops.